static/src/js/util/request/logoutRequest.js

```diff
@@ -0,0 +1,18 @@
+import Request from './request'
+import { getEnvironmentConfig } from '../../../../../sharedUtils/config'
+
+/**
+ * Request object for concept specific requests
+ */
+export default class LogoutRequest extends Request {
+  constructor(authToken) {
+    super(getEnvironmentConfig().apiHost)
+
+    this.authenticated = true
+    this.authToken = authToken
+  }
+
+  logout() {
+    return super.delete('/logout')
```

Comment:
We've used `this.` for all of our other request classes, is there a difference in this call?

Reply:
the one I copied from uses super 🤷‍♂ probably change this one and conceptRequest.js